integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/monitoring/AlarmRegister.feature

```diff
@@ -24,13 +24,13 @@ Feature: SmartMetering Monitoring - Alarm Register
     Then the alarm register should be returned
       | DeviceIdentification | TEST1024000000001 |
 
-  Scenario: Clear alarm register SMR 5.1
+  Scenario Outline: Clear only alarm register 1 for these protocols
```

Comment:
` <protocol> en <version> opnemen in de Scenario beschrijving, dan zie je welke test faalt`

Reply:
Aangepast